src/cockpit/channels/dbus.py

```diff
@@ -289,7 +289,7 @@ def sync_handler(message):
 
     async def do_call(self, message):
         path, iface, method, args = message['call']
-        timeout = message.get('timeout')
+        timeout = message.get('timeout', 2**63)
```

Comment:
`2**64 - 1`, yes?

Not that it matters.  Just looks weird.

Reply:
Yes, indeed.